userspace/pyproject.toml

```diff
@@ -39,7 +40,7 @@ dependencies = [
   "pydub",
   "pyjwt",
   "pylint",
-#"pyopencl == 2024.1", #FIXME: has some installation errors
+  "pyopencl == 2024.1",
```

Comment:
any idea why we have to pin? this is just a temp fix

Reply:
Any newer version also needs a newer version of cmake to compile. Cmake >= 3.17 is not available via apt in 20.04. We could pull it in via pip.

Reply:
that's fine, let's just leave a comment

Reply:
You mean keep it pinned for now? We can probably unpin it with 24.04 later.

Reply:
yeah, but leave a comment about why it's pinned